awx/ui/client/lib/components/tag/_index.less

```diff
@@ -8,6 +8,7 @@
     flex-direction: row;
     align-content: center;
     min-height: @at-space-4x;
+    max-height: @at-space-4x;
```

Comment:
what if tag names are long and take up multiple lines?

Reply:
They'll get truncated - I think they should show ellipses though as well, what do you think?

Reply:
At a minimum, we don't want this to apply to all tags so I've moved this change to the wrapper class used for input field tags.